starter-code/src/app/app.component.html

```diff
@@ -1,3 +1,9 @@
-<h1>
-  {{title}}
-</h1>
+<header>
+  <p>Nice header</p>
+  <a [routerLink]="['/home']">Home</a>
+  <a [routerLink]="['/movie', movie]">Details</a>
```

Comment:
`<a [routerLink]="['/movie', movie]">Details</a>` should be on the homePage passing the movie's id with `movie.id`